docs/I2CDEVICES.md

```diff
@@ -42,8 +42,8 @@ Index | Define              | Driver  | Device   | Address(es) | Description
   19  | USE_SI1145          | xsns_24 | SI1147   | 0x60        | Ultra violet index and light sensor
   20  | USE_LM75AD          | xsns_26 | LM75AD   | 0x48 - 0x4F | Temperature sensor
   21  | USE_APDS9960        | xsns_27 | APDS9960 | 0x39        | Proximity ambient light RGB and gesture sensor
-  22  | USE_MCP230xx        | xsns_29 | MCP23008 | 0x20 - 0x26 | 16-bit I/O expander
-  22  | USE_MCP230xx        | xsns_29 | MCP23017 | 0x20 - 0x26 | 16-bit I/O expander
+  22  | USE_MCP230xx        | xsns_29 | MCP23008 | 0x20 - 0x27 |  8-bit I/O expander
+  22  | USE_MCP230xx        | xsns_29 | MCP23017 | 0x20 - 0x27 | 16-bit I/O expander
```

Comment:
There is a good reason that 0x27 is NOT listed
0x27 has been excluded to avoid conflict with some other devices such as the I2C LCD display
From the driver's code:
```
/*********************************************************************************************\
   MCP23008/17 - I2C GPIO EXPANDER

   Docs at https://www.microchip.com/wwwproducts/en/MCP23008
           https://www.microchip.com/wwwproducts/en/MCP23017

   I2C Address: 0x20 - 0x26 (0x27 is not supported)
\*********************************************************************************************/
```

Reply:
but it is supported and therefore wrong to state range from 0x20 - 0x26. Write a comment to this table or reference to MCP page; but it is confusing to write different ranges on different pages.

Reply:
Working doesn't mean supported.
Someone in Tasmota team decided that this value should be excluded to avoid conflicts with other device.
Please remove this change from your PR

Reply:
changed those lines back